controller/service.rst

```diff
@@ -12,15 +12,9 @@ injection like any other normal service.
 Referencing your Service from Routing
 -------------------------------------
 
-Registering your controller as a service is great, but you also need to make sure
-that your routing references the service properly, so that Symfony knows to use it.
-
-If the service id is the fully-qualified class name (FQCN) of your controller, you're
-done! You can use the normal ``AppBundle:Hello:index`` syntax in your routing and
-it will find your service.
-
-But, if your service has a different id, you can use a special ``SERVICEID:METHOD``
-syntax:
+In addition to registering your controller as a service, you also need to make
+sure that your routing references the service properly, so that Symfony can find
+it. Use the ``service_id:method_name`` syntax:
```

Comment:
In the example below, it's still referring to `@Route(service="app.hello_controller")` from the FrameworkExtraBundle. In 3.3 (correct me if I'm wrong), it's not longer required to do this when you have the FQCN as service id and using the core framework `@Route()` annotation on methods.

Would it be an idea to simplify this to the FQCN? While it's still possible to register services this way, I would not recommend it in the documentation.